lib/vdsm/storage/lvm.py

```diff
@@ -1439,6 +1439,26 @@ def createVG(vgName, devices, initialTag, metadataSize, force=False):
         _lvminfo._invalidatevgs(vgName)
         log.debug("Cache after createvg %s", _lvminfo._vgs)
 
+    # Disable autoactivation after creating a VG
+    _disable_autoactivation_vg(vgName)
```

Comment:
Doing it here makes sense, this is part of vg creation flow, and it should happen
before we create any lv on this vg.

The comment is not needed, it does not add any value. It can add value if
it will describe why this is done at this point (before any lv can be created).

There is an issue with error handling - if creating the vg succeeded, but disabling
autoactivation failed, we raise se.VolumeGroupCreateError. From caller point of view
creating the vg failed, so it may not try to clean up after the failure, and we are
left with lefover vg using the pv. This may break engine create vg flow and require
manual cleanup.

Please check if this can be integrated in a higher level, so if creating and 
configuring a vg fails no leftovers are left.